flocker/acceptance/testtools.py

```diff
@@ -157,6 +151,10 @@ def _clean_node(test_case, node):
     # http://doc-dev.clusterhq.com/advanced/cleanup.html#removing-zfs-volumes
     # A tool or flocker-deploy option to purge the state of a node does
     # not yet exist. See https://clusterhq.atlassian.net/browse/FLOC-682
+    containers = _run_SSH(22, 'root', node, ['docker', 'ps', '-aq'], None)
```

Comment:
http://doc-dev.clusterhq.com/advanced/cleanup.html says that to achieve this, three steps must be performed (in order):
* Stop Containers
* Remove Containers
* Remove ZFS Volumes

Please either change this to exactly match those docs' commands, or explain why not.

Also, I get that this messy option is faster and we do it a lot in these scripts, but if running `flocker_deploy` with "clean" configurations does the same job but slower, can we not replace those docs with an instruction to run `flocker_deploy` with "clean" configurations.

In summary, I think that the code should match and link to the docs, or have comments explaining why it doesn't.